amazon-dynamodb-quickstart/src/main/docker/Dockerfile.native

```diff
@@ -1,22 +1,27 @@
 ####
 # This Dockerfile is used in order to build a container that runs the Quarkus application in native (no JVM) mode
 #
-# Before building the docker image run:
+# Before building the container image run:
 #
-# mvn package -Pnative -Dquarkus.native.container-build=true
+# ./mvnw package -Pnative
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile.native -t quarkus/dynamodb-client .
+# docker build -f src/main/docker/Dockerfile.native -t quarkus/amazon-dynamodb .
 #
 # Then run the container using:
 #
-# docker run -i --rm -p 8081:8081 quarkus/dynamodb-client
```

Comment:
I wonder if there is a reason why `8081` was being used

Reply:
Yeah, no idea :).

The README talks about 8080 so I think we should be good.